### DISCUSSION TITLE
What is a numerical subordinate user ID?

### DISCUSSION THREAD

**Discussion by truh** [containers/podman#23934] [Q&A] [answered]
And why is it 10000 in the manual? (`sudo usermod --add-subuids 10000-75535 USERNAME`)

**Comment by eriksjolund** [comment, marked as answer]
Regarding the number _10000_ here

* https://github.com/containers/podman/blob/c66d46c39ad7f6560da8f276d087cf9d033fc4b2/docs/source/markdown/podman.1.md?plain=1#L441

The chosen number would typically be the lowest unused subuid. 
The chosen range must not overlap with any of the existing ranges in _/etc/subuid_  (or any subuid ranges that are configured via /etc/nsswitch.conf)

Fortunately you often don't need to use __usermod --add-subuids__ very often because __useradd__ configures subuids and subgids automatically for a new user. (At least when running fedora).

This document
https://wiki.gentoo.org/wiki/Subuid_subgid
has a short introduction to the topic of subuids and subgids.

**Reply by truh** [reply]
Thanks the gentoo wiki is a lot more helpful on this matter than the subgid and podman man pages.

So the numbers in the command is and actual range but in the /etc files it's a starting number and an offset? That's not very ergonomic.

> Fortunately you often don't need to use usermod --add-subuids very often because useradd configures subuids and subgids automatically for a new user. (At least when running fedora).

Seems like I'm not so fortunate on RHELL8.
